src/routes/notifications/v1/entities/__tests__/create-signature.builder.ts

```diff
@@ -0,0 +1,18 @@
+import type { UUID } from 'crypto';
+import { generatePrivateKey, privateKeyToAccount } from 'viem/accounts';
+
+export async function safeRegistrationSignatureBuilder(args: {
+  signaturePrefix: string;
```

Comment:
Although it won’t change, I would still keep it as an object property for the sake of clean code and flexibility.

Reply:
In which case, I'd suggest adding a comment to the relevant Transaction Service implementation. It's otherwise possible to make the signature incorrect by passing a prefix.

Reply:
My suggestion is to leave it as it is since it's test and also we will eventually remove the v1. It's also inside tests so for that reason it doesn't matter if we hardcode it in the test or in the builder.